controlplane/kubeadm/internal/workload_cluster_etcd.go

```diff
@@ -66,27 +66,23 @@ loopmembers:
 }
 
 // UpdateEtcdLocalInKubeadmConfigMap sets etcd local configuration in the kubeadm config map.
-func (w *Workload) UpdateEtcdLocalInKubeadmConfigMap(etcdLocal *bootstrapv1.LocalEtcd) func(*bootstrapv1.ClusterConfiguration) {
+func (w *Workload) UpdateEtcdLocalInKubeadmConfigMap(etcdLocal bootstrapv1.LocalEtcd) func(*bootstrapv1.ClusterConfiguration) {
```

Comment:
Not sure about this one, same for UpdateEtcdExternalInKubeadmConfigMap

(+ corresponding unit tests cases I dropped)